packages/rocketchat-livechat/server/hooks/sendToCRM.js

```diff
@@ -49,6 +58,31 @@ RocketChat.callbacks.add('livechat.saveInfo', (room) => {
 	return sendToCRM('LivechatEdit', room);
 }, RocketChat.callbacks.priority.MEDIUM, 'livechat-send-crm-save-info');
 
+RocketChat.callbacks.add('afterSaveMessage', function(message, room) {
+	// only call webhook if it is a livechat room
+	if (!(typeof room.t !== 'undefined' && room.t === 'l' && room.v && room.v.token)) {
+		return message;
+	}
+
+	// if the message has a token, it was sent from the visitor
+	// if not, it was sent from the agent
+	if (message.token) {
+		if (!RocketChat.settings.get('Livechat_webhook_on_visitor_message')) {
+			return message;
+		}
+	} else if (!RocketChat.settings.get('Livechat_webhook_on_agent_message')) {
+		return message;
+	}
+
+	// if the message has a type means it is a special message (like the closing comment), so skips
+	if (message.t) {
+		return message;
+	}
+
+	sendToCRM('Message', room, [message]);
+	return message;
+}, RocketChat.callbacks.priority.MEDIUM, 'sendMessageToFacebook');
```

Comment:
The callback ID/name is wrong `sendMessageToFacebook`

Reply:
@rodrigok, fixed callback ID to `livechat-send-crm-message`